opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -280,16 +300,75 @@ def export(
                     UNIQUE_IDENTIFIER_KEY
                 ] = self.unique_identifier
 
-            point = series.points.add()
-            if isinstance(record.aggregator, SumAggregator):
-                data_point = record.aggregator.checkpoint
-            elif isinstance(record.aggregator, ValueObserverAggregator):
-                data_point = record.aggregator.checkpoint.last
+            point_dict = {"interval": {}}
+
+            if isinstance(record.aggregator, HistogramAggregator):
+                count = sum(record.aggregator.checkpoint.values())
+                buckets = list(record.aggregator.checkpoint.items())
+
+                point_dict["value"] = {
+                    "distribution_value": Distribution(
+                        count=count,
+                        bucket_counts=[bucket[1] for bucket in buckets],
```

Comment:
Perhaps do the computation of bucket_counts and bucket bounds outside of here (maybe using a `zip`), its a little confusing for the reader who doesn't know what bucket[1] and bucket[0] represent.

Reply:
done